src/components/GamepadController.js

```diff
@@ -0,0 +1,33 @@
+import React, { useContext, useEffect, useState } from 'react';
+
+import Fab from '@material-ui/core/Fab';
+import Gamepad from '@material-ui/icons/Gamepad';
```

Comment:
Why not import { Fab, Gamepad } from '@material-ui/icons';?

Reply:
That would create a much bigger client payload this way you only add used icons to the final product